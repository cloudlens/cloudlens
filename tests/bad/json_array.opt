-format json -jsonpath obj.field
