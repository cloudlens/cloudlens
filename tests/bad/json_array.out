Parse Error: the log stream obj.field must be a json Array
